java-client/session/src/main/java/io/deephaven/client/impl/UnknownObject.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.client.impl;
+
+import io.deephaven.client.impl.ObjectService.Bidirectional;
+import io.deephaven.client.impl.ObjectService.Fetchable;
+
+/**
+ * A server object that is <b>not</b> {@link Fetchable} nor {@link Bidirectional}; the server does not have a
+ * registration for the underlying object.
```

Comment:
What does "the server does not have a registration for the underlying object" mean (to a user)?